Sources/xcodeproj/Objects/Project/PBXOutputSettings.swift

```diff
@@ -0,0 +1,112 @@
+//
```

Comment:
Sorry for the nitpick, but would you mind removing the comments? 😬

Reply:
Oops. Will do.

Reply:
I think you missed this comment @drekka